traffic_ops/traffic_ops_golang/deliveryservice/servers/delete.go

```diff
@@ -45,10 +45,11 @@ func DeleteDeprecated(w http.ResponseWriter, r *http.Request) {
 }
 
 const lastServerQuery = `
-SELECT COUNT(*) = 0
-FROM deliveryservice_server
-WHERE deliveryservice = $1
-	AND server <> $2
+	SELECT COUNT(*)
```

Comment:
Why remove the `= 0`? The query is expected to return a boolean value not an actual count. As-is, does this actually scan successfully into a bool?

Reply:
`= 0` gives true even when it shouldn't. So I replaced it with` > 0`